tests/fsharp/core/tests_core.fs

```diff
@@ -676,6 +682,8 @@ module Printing =
     let printing flag diffFileOut expectedFileOut diffFileErr expectedFileErr = check (attempt {
         let { Directory = dir; Config = cfg } = testContext ()
 
+        do! requireENCulture ()
```

Comment:
yes, it's temporary, until `--preferreduilang` is merged. After that i reenable tests,
meanwhile the appveyor/jenkins run tests anyway, so no issues.